### ISSUE TITLE
Documentation: Static site Fail On CI

### ISSUE THREAD

**Issue by JustinDSN** [square/workflow#946]
The Travis CI job "Documentation: Static site" fails because mkdocs-material 4.6.1 depends on markdown >= 3.2 which requires python >= 3.5.

I'll bump the python version in a sample PR to make sure it works.

```
0.01s$ source ~/virtualenv/python2.7/bin/activate
$ python --version
Python 2.7.15
$ pip --version
pip 19.0.3 from /home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/pip (python 2.7)
0.65s$ pip install mkdocs-material
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
Collecting mkdocs-material
  Downloading https://files.pythonhosted.org/packages/4c/3a/b3ea73d570045a1e660cb7a531b07c585fbd872712aa6e6c9b44d5d0fbb0/mkdocs_material-4.6.1-py2.py3-none-any.whl (722kB)
Collecting markdown>=3.2 (from mkdocs-material)
  Could not find a version that satisfies the requirement markdown>=3.2 (from mkdocs-material) (from versions: 1.7, 2.0, 2.0.1, 2.0.2, 2.0.3, 2.1.0, 2.1.1, 2.2.0, 2.2.1, 2.3, 2.3.1, 2.4, 2.4.1, 2.5, 2.5.1, 2.5.2, 2.6, 2.6.1, 2.6.2, 2.6.3, 2.6.4, 2.6.5, 2.6.6, 2.6.7, 2.6.8, 2.6.9, 2.6.10, 2.6.11, 3.0, 3.0.1, 3.1, 3.1.1)
No matching distribution found for markdown>=3.2 (from mkdocs-material)
The command "pip install mkdocs-material" failed and exited with 1 during .
Your build has been stopped.
```

**Comment by JustinDSN** [comment]
Bumped pyton version here: https://github.com/square/workflow/pull/925/commits/c28c3ad6776b89fa281ded561048e0ae37b52258

Travis job here:
https://travis-ci.com/square/workflow/jobs/285162431

**Comment by JustinDSN** [comment]
Resolved by #925
